kernel/interrupts.c

```diff
@@ -32,12 +32,12 @@ void interrupt_new_handler(int intnum, void (*handler)(interrupt_frame*)) {
 
 void interrupt_init() {
     /* ICW1 - begin initialization */
-	outb(0x20, 0x11);
+    outb(0x20, 0x11);
     outb(0xA0, 0x11);
 
     /* ICW2 - remap offset address of IDT */
     outb(0x21, 0x20);
-    outb(0xA1, 0x82);
+    outb(0xA1, 0x28);
```

Comment:
waarom dit?

Reply:
omdat er een typo zat in dat getal en 0x82 totaal niet logisch is (0x28 = 0x20 + 8)